README.md

```diff
@@ -22,6 +22,14 @@ To install, run the following (assuming composer is in your PATH)
 ```sh
 composer require cleanse/discord-hypertext
```

Comment:
should change this to `composer require cleanse/discord-hypertext dev-master` while you are editing this file